generator/translator_planet.cpp

```diff
@@ -95,6 +96,9 @@ void TranslatorPlanet::EmitElement(OsmElement * p)
       break;
 
     ft.SetOsmId(base::MakeOsmWay(p->id));
+
+    m_maxspeedCollector.Process(*p);
```

Comment:
Please move this to m_routingTagsProcessor.

Reply:
Done